### DISCUSSION TITLE
Z-Offset is increasing by 0.05 After deploying the firmware

### DISCUSSION THREAD

**Discussion by KatjaWfromG** [mriscoc/Ender3V2S1#1216] [Q&A]
Hi,

I have 
Ender 3 V2
Board 4.2.7
Dual Z-Axis
CR Touch
Spider Hotend 3.0
everytime when I deploy a Version of the Professional Firmware the Z-Offset increases by 0.05.
I have actually deployed the Version from December.
This behaviour is reproducable.

What am I doing wrong?
Hope someone can help me.

**Comment by mriscoc** [comment]
A value of 0.05 is within the margins of error.

**Reply by KatjaWfromG** [reply]
Ok. But why is it increasing by 0.05 every time I deploy this image? When I started my z-Offset was 1.72, meanwhile I am at 1.92.
